src/main/java/Task/Task.java

```diff
@@ -0,0 +1,56 @@
+package task;
+
+import java.time.LocalDateTime;
+
+/**
+ * Task is the main functionality of Duke. It is the unit that information given by the user
+ * is stored as in Duke and is also how information from Duke is read to the user. It is an
+ * abstract class and can only be represented as a Deadline, Event or Todo_task.
+ *
+ * @author Joshua
+ */
+public abstract class Task {
+    /**
+     * position is the location of the task in the TaskList.
+     * taskDescription is the contents of the task.
+     * taskCompleted is the indicator of whether the task has been done or not.
+     * date is the scheduled time for the task to occur.
+     */
+    protected int position;
+    protected String taskDescription;
+    protected boolean taskCompleted;
```

Comment:
Should this boolean be named to sound more like a boolean instead (e.g. isCompleted)?

Reply:
Thank you, changed it.